### ISSUE TITLE
generalization performance is bad

### ISSUE THREAD

**Issue by shengc** [shengc/tf-han#2]
I have corrected an implementation error that did not treat sentences from the same document as a coherent sequence. However the generalization performance on validation set remains to be bad.

**Comment by zZyan** [comment]
Hi, 

I noticed that for 'words_attention' and 'sentence_attention', inputs params for attention layer are different. I suspect in both case, `hidden` should be passed in. 

i.e.
```      
with tf.variable_scope('words_attention'):
    hidden = tf.layers.dense(outputs, units=self._hidden_dim * 2, activation=tf.nn.tanh, kernel_initializer=tf.random_uniform_initializer(-0.1, 0.1))
    attention =  tf.layers.dense(hidden, units=1,  activation=None, kernel_initializer=tf.random_uniform_initializer(-0.1, 0.1))

**Comment by shengc** [comment]
@zZyan you are more than welcome to send a PR :-)
